src/content/docs/errors-inbox/version-tracking.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Error tracking with errors inbox'
+metaDescription: "Errors inbox is an error tracking solution designed to give you the tools to proactively detect, triage, and resolve errors across your full application stack."
+---
+
+import errorsinboxFilterErrorsbyVersions from 'images/errors-inbox_screenshot-crop_filter-errors-by-versions.webp'
+
+Errors inbox version tracking allows developers to monitor each deployment to check for accuracy and success. Lets say you and your team an alert that says one of your entities has a spike in error rates. You identify the [source of the error](docs/tutorial-error-tracking/respond-outages/) and decide to release a canary deployment. You can use version tracking to monitor your fix as well as the previous deployments to determine if you solved the problem or if you need to go back to the drawing board. 
+
+<Steps>
+<Step>
+## Instrument version tracking on your entities. 
```

Comment:
This section was to tell the user how the fields they will instrument are used in Errors inbox and then the 'configure fields' section below tells them how to instrument them. It is helpful for them to know what order we use the fields so they will understand  if they have `service.version` and `tags.commit `both set - we will use `service.version`.

As it is currently written, it seems repetitive and doesn't give them that key info on how we will use the fields.